File: CECVS_Vacay/CECVS.Vacay.Domain/Models/Ferias.cs
using System;
using System.Collections.Generic;

namespace CECVS.Vacay.Domain.Models
{
    public partial class Ferias
    {
        public int IdFerias { get; set; }
        public int IdFuncionario { get; set; }
        public DateTime DtInicio { get; set; }
        public int QtDias { get; set; }

        public virtual Funcionario IdFuncionarioNavigation { get; set; } = null!;
    }
}

File: CECVS_Vacay_Front/src/models/ferias.ts

interface IFeriasDTO {
    idFerias: number,
    idFuncionario: number,
    dtInicio: string,
    qtDias: number
}

export type { IFeriasDTO };

interface IFeriasUpdateDTO {
    idFerias: number,
    dtInicio: string,
    qtDias: number
}

export type { IFeriasUpdateDTO };


interface IFeriasExpandidoDTO {
    idFerias: number,
    idFuncionario: number,
    coMatricula: string,
    noFuncionario: string,
    dtInicio: string,
    qtDias: number
}

export type { IFeriasExpandidoDTO };